app/models/dialog_field.rb

```diff
@@ -13,6 +13,7 @@ class DialogField < ApplicationRecord
                                   :message => "Field Name %{value} is reserved."}
 
   default_value_for :required, false
+  default_value_for :visible, true
```

Comment:
I don't know, I would have assumed a nil value would be `true` when you initialize the object on find or create but maybe not.  I don't remember.